openstack/keystone/templates/etc/_ratelimit.yaml.tpl

```diff
@@ -0,0 +1,99 @@
+# List of whitelisted scopes keys (domainName/projectName).
+whitelist:
+  - Default/service
+  - monsoon3/cc-demo
+  - cc3test/admin
+  - ccadmin/cloud_admin
+
+# Override default ratelimit response.
+ratelimit_response:
+  status: 429 Rate Limited
+  status_code: 429
+  body: "Rate Limit Exceeded"
+
+# Override default blacklist response.
+blacklist_response:
+  status: 497 Blacklisted
```

Comment:
> the naming convention is taken from here : https://github.com/sapcc/openstack-rate-limit-middleware/blob/master/docs/configure.md#black---whitelist

I understand it, but we have a more general naming convention: https://sapedia.one.int.sap/wiki/Blacklist

I am fine with the internal names, like `blacklist_response`, because it is seen by us only, but customer-facing responses should be in line with the general SAP conventions.